<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 54px;
background: #FFF;</string>
  </property>
  <widget class="QPushButton" name="Path_Button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>368</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 13px;
background: #1C1C1C;
color: #FFF;
font-family: Poppins;
font-size: 12px;
font-style: normal;
font-weight: 700;
line-height: normal;</string>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Confirm_Button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 13px;
border: 1px solid #1C1C1C;</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-60</y>
     <width>871</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Smooch Sans</family>
          <pointsize>-1</pointsize>
          <weight>87</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #1C1C1C;
leading-trim: both;
text-edge: cap;
font-family: Smooch Sans;
font-size: 60px;
font-style: normal;
font-weight: 700;
line-height: normal;</string>
        </property>
        <property name="text">
         <string>Enter your token</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Token">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 13px;
background: #F2F2F2;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Smooch Sans</family>
          <pointsize>-1</pointsize>
          <weight>87</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #1C1C1C;
leading-trim: both;
text-edge: cap;
font-family: Smooch Sans;
font-size: 60px;
font-style: normal;
font-weight: 700;
line-height: normal;</string>
        </property>
        <property name="text">
         <string>Download path</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Path">
        <property name="styleSheet">
         <string notr="true">color: #1C1C1C;
font-family: Poppins;
font-size: 20px;
font-style: normal;
font-weight: 400;
line-height: normal;</string>
        </property>
        <property name="text">
         <string>PATH</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
